bin/s3Create.js

```diff
@@ -2,5 +2,5 @@ const s3Create = require('../lib/s3Create')
 
 const file = process.argv[2]
 s3Create(file)
-  .then(console.log)
-  .catch(console.error)
```

Comment:
This was actually just our node script for testing our s3Create function. It's not really needed anymore, but in this case the console.log/console.error are fine.